style.css

```diff
@@ -0,0 +1,92 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  background: #f7f7f9;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 52px;
+  color: #00f;
+  margin: 0;
+}
+
+h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin: 0;
+}
+
+p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.menu_btn {
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+  width: 45px;
+}
+
+.header-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 14px 24px;
+  background-color: white;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-height: 90vh;
```

Comment:
- [ ] **[OPTIONAL]:** I would suggest that you use the CSS's `calc` function for this section height. Since you need it to be the entire screen minus the height of the toolbar, this function comes in very handy: 🤓 🛠️ 

```suggestion
  min-height: calc(100vh - 81px); 
```
💡 _Note: the 81px comes from the actual height of the toolbar as shown below._

![image](https://user-images.githubusercontent.com/77069011/211868499-c6448679-0bba-40e5-aa4d-bee000af0a49.png)